src/core/server/saved_objects/mappings/lib/get_root_type.ts

```diff
@@ -18,20 +18,23 @@
  */
 
 import { getTypes } from './get_types';
+import { EsMappings } from './types';
 
 /**
- *  Get the singular root type in the EsMappingsDsl
+ *  Get the singular root type in the EsMappings
  *  object. If there are no types, or there are more
  *  that one type, this function will throw an error.
  *
- *  @param  {EsMappingsDsl} mappings
+ *  @param  {EsMappings} mappings
  *  @return {string}
  */
-export function getRootType(mappings) {
+export function getRootType(mappings: EsMappings): string {
```

Comment:
Added a type here. Rest are prettier changes.

Reply:
I'd rather we relied on the actual function implementation to define the return type, TypeScript should be able to infer that this function returns a `string`, and if it can't I think we need to update the function implementation, not explicitly define a return type.